doc/languages.rst

```diff
@@ -319,6 +319,10 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
          A list of library directories.
 
+   .. syntax-checker:: erlang-rebar3
+
+      Check Erlang with the `rebar3 <https://www.rebar3.org/>`_ build tool.
```

Comment:
Please also add a paragraph right below `..supported-language:: erlang` that explains the order in which syntax checkers are applied, e.g. ``Flycheck checks erlang with `erlang-rebar3` in rebar projects or `erlang` otherwise.``